Multisig/UI/App/ViewControllerFactory.swift

```diff
@@ -22,7 +23,20 @@ enum ViewControllerFactory {
             nav.modalPresentationStyle = .fullScreen
             nav.modalTransitionStyle = .crossDissolve
 
-            let start = LaunchView(acceptedTerms: .constant(false), onStart: { [weak nav] in
+            let start = LaunchView(acceptedTerms: .constant(false), onStart: { [weak nav] trackingEnabled in
+                AppSettings.trackingEnabled = trackingEnabled
+                if trackingEnabled {
+                    // https://firebase.google.com/docs/ios/supporting-ios-14
+                    // This is required only for IDFA that we do not use
+                    if #available(iOS 14, *) {
+                        ATTrackingManager.requestTrackingAuthorization { status in
```

Comment:
in this case, I would expect that if the user says "Ask app not to track", then we need to disable tracking, even if user pressed "Agree" button. Do you think this is correct?

Reply:
This is what I did initially, but then if to read system message on the alert it says about tracking _across_ different apps/sites that is not related to tracking within the app

Reply:
the point is that we share the tracking information and it travels from the device to other servers, so Apple treats it as tracking and that is what expected case. I think it is misleading if user disallows and we still keep the "tracking" enabled, even if the user tried to enable the tracking before

Reply:
This was my way of thinking too before I dug into more details. If google tracking should be bound to this permission, then it would be logical to make this check within google lib. 
After reading some info (e.g. here https://firebase.google.com/docs/ios/supporting-ios-14) - "Analytics event logging, event reporting, and conversion measurement are unaffected, but attribution is impacted if IDFA is not accessible."
Taking into account apple system message stating only cross-apps tracking I did not bind them. So this permission is only about sharing IDFA info.

Reply:
@tschubotz please let us know what do you think we should do

Reply:
This is inconsequential, with this reasoning, why do we have to use this tracking manager at all, if it only required for IDFA tracking?

Reply:
By the Apple's User Privacy and Data Use, `Tracking also refers to sharing user or device data with data brokers`, so we must ask permission with this alert.

However, if user chooses "Ask App Not To Track", the Firebase Analytics event collection is not affected, i.e. we still get anonymized data. 

In my opinion, this situation - if user selects that option but we still have analytics enabled - is going to be confusing for the user. The natural question is: why does the app still has tracking enabled even if I asked the app not to track.

So the user's choice would be not to track.

For that reason, we should disable the tracking if that choice was given

Reply:
hmmm. After reading https://github.com/firebase/firebase-ios-sdk/issues/5928, I think as long as this IDFA not set, they don't match the user's data on their, no? Hence we perhaps don't even need this popup?

Reply:
IMO this require more product thinking.
If to follow this logic, then on terms screen I would remove "Agree without tracking" button and would alway request this dialog. If user says "no" later she can enable it in advanced settings, but this will be not related to this permission. And what to do if user goes to phone settings and switches tracking off for the app (like with push notifications)?

Reply:
ok, after discussion:

- We need to have one button on terms screen saying "Agree" and the other one saying "No, thanks" (or what it was before?) to close the "app terms" sheet
- After agreeing, we present the system App Tracking alert and depending on user choice:
  - if "Allow" - then enable tracking and enable crashlytics
  - if "Ask App Not To Track" - then do not enable tracking or crashlytics
- In the advanced app settings, we may need to rephrase it to "Share Usage and Crash Data" instead of "Share Usage Data"
- For existing users updating to iOS 14.5, the app should ask the system permission with the same logic as above.
- If user changes the permission in the phone/device settings, then the app should disable tracking if the user denied the permission. If the user changes permission to allow tracking in the device, then the app can start tracking only if it is also enabled in the app settings.

Reply:
For users on iOS 13, things should still work as before?